tests/dockerConfiguration/publicAddressVariable_test.js

```diff
@@ -0,0 +1,109 @@
+const assert = require('assert');
+
+Feature('Tests for PMM_PUBLIC_ADDRESS environment variable');
+
+const dockerVersion = process.env.DOCKER_VERSION || 'perconalab/pmm-server:dev-latest';
+let portalUser;
+let adminToken;
+let freeOrg;
+const publicIPs = new DataTable(['testCase', 'publicAddress']);
+
+publicIPs.add(['PMM-T1173', '127.0.0.1']);
+publicIPs.add(['PMM-T1173', '127.0.0.1:8443']);
+publicIPs.add(['PMM-T1174', 'ec2-18-188-74-98.us-east-2.compute.amazonaws.com']);
```

Comment:
the host value I assume is just for testing purpose, it doesn't really come from the worker?

Reply:
Yes it is just for testing since we need to test more versions